Fastfile

```diff
@@ -272,6 +273,44 @@ reflected in `Info.plist` during building."
 
   end
 
+  desc "Generates the push notifications certificates for the build configurations mapped to `:test` and `:qa`."
+  lane :generate_push_certificates_development do |options|
+    generate_push_certificates build_configuration: :test
+    generate_push_certificates build_configuration: :qa
+  end
+
+  desc "Generates the push notifications certificates for the build configurations mapped to `:appstore`."
+  lane :generate_push_certificates_appstore do |options|
+    generate_push_certificates build_configuration: :appstore
+  end
+
+  desc "Generates the push notifications certificates for the build configuration provided."
+  desc "It uses the same password stored in keychain for the current user."
+  desc "Make sure to store safely the output of this command in the right `Google Drive` folder."
+  private_lane :generate_push_certificates do |options|
+    build_configuration = options[:build_configuration]
+
+    desc "Bundle identifier from xCode project"
+    bundle_identifier = read_project_property(
+      build_configuration: build_configurations[build_configuration],
+      build_setting: 'PRODUCT_BUNDLE_IDENTIFIER'
+    )
+
+    desc "Password for current user from keychain"
+    user = CredentialsManager::AppfileConfig.try_fetch_value(:apple_id)
```

Comment:
Should we show an error message in case this can't be found?

Reply:
`Fastlane` itself will ask for it.